saged/utils.py

```diff
@@ -437,14 +438,48 @@ def load_recount_data(config_file: str) -> Tuple[pd.DataFrame, Dict[str, str], D
     expression_df = load_compendium_file(compendium_path).T
     sample_to_study = recount_map_sample_to_study(metadata_path)
 
-    # Once I implement txt2onto, this will call a function that actually stores things in the dict
     sample_to_label = None
     with open(dataset_config.pop('label_path'), 'rb') as in_file:
         sample_to_label = pickle.load(in_file)
 
     return expression_df, sample_to_label, sample_to_study
 
 
+@functools.lru_cache()
+def load_biobert_embeddings(config_file: str) -> np.array:
+    """
+    Read the biobert embeddings of the metadata for each sample in the dataset
+
+    Arguments
+    ---------
+    config_file: The file containing paths to the embedding file
+
+    Returns
+    -------
+    biobert_embeddings - A numpy array containing the biobert embeddings of the metadata in the
+                         same order as expression_df
```

Comment:
```suggestion
                         same sample order as expression_df
```
I think this is accurate? Since you want to generate different features to use in addition to the genes that you already have, but they should cover the same samples in the same order?

Reply:
Yeah that's definitely more clear